Corona-Warn-App/src/main/java/de/rki/coronawarnapp/familytest/core/storage/FamilyTestDatabase.kt

```diff
@@ -99,7 +99,17 @@ interface FamilyCoronaTestDao {
     suspend fun update(identifier: TestIdentifier, update: (FamilyCoronaTest) -> FamilyCoronaTest) {
         get(identifier)?.let {
             val updated = update(it.test).toEntity()
-            insert(updated)
+            if (it != updated) insert(updated)
+        }
+    }
+
+    @Transaction
```

Comment:
Change the param updates to (List<FamilyCoronaTest>) -> List<FamilyCoronaTest>
Would be much more pratical I guess.

Reply:
I thought about it. It seems harder that way. How would I come up with a function that modifies the whole list?

Reply:
its only a suggestion. Feel free to merge if its to complex in your eyes.